src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -766,6 +766,30 @@ class ResourceServiceSpec extends FlatSpec with Matchers with ScalaFutures with
     assert(policyDAO.listAccessPolicies(parentResource, samRequestContext).unsafeRunSync().nonEmpty)
   }
 
+  "validatePolicy" should "succeed with a correct policy" in {
+    runAndWait(service.validatePolicy(defaultResourceType, service.ValidatableAccessPolicy(AccessPolicyName("a"), Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject])), Set(ResourceRoleName("owner")), Set(ResourceAction("alter_policies")), Set()))) shouldBe empty
```

Comment:
nit:  I find long one liners like this harder to grok and figure out which parens match up where.

```suggestion
    val emailToMaybeSubject = Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject]))
    val policy = service.ValidatableAccessPolicy(AccessPolicyName("a"), emailToMaybeSubject, Set(ResourceRoleName("owner")), Set(ResourceAction("alter_policies")), Set())
    runAndWait(service.validatePolicy(defaultResourceType, policy)) shouldBe empty
```